tool/python/examples/train_cifar10_2.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
can we put the loop code as comments in train_cifar10.py (most of the code is the same as that)?